Plotting offers to select uninitialized ensembles leading to an error

### What happened? (You can include a screenshot if it helps explain)

When creating an empty experiment and unitilized ensemble, it shows it plotting window, which then shows an error:
```
 [ERROR] ert.dark_storage.endpoints.parameters: 'No SCALAR dataset in storage for ensemble ensemble'
```

<img width="1380" height="733" alt="Image" src="https://github.com/user-attachments/assets/0c1cb391-a07a-44f6-90a3-e3faccd97521" />

### What did you expect to happen?

The ensemble shouldn't be listed in plotting window when being not initialized.

### Steps to reproduce


`ert gui poly.ert`
Manage experiments panel -> create new experiment -> create plot -> select ensemble

```Python

```

### Environment where bug has been observed

- [x] Python 3.11
- [x] Python 3.12
- [x] macOS
- [x] RHEL8
- [x] RHEL9
- [x] Local queue
- [x] LSF queue
- [x] Slurm queue
- [x] OpenPBS queue

--- a/src/ert/dark_storage/endpoints/ensembles.py
+++ b/src/ert/dark_storage/endpoints/ensembles.py
@@ -1,4 +1,5 @@
 import logging
+from collections import Counter
 from uuid import UUID
 
 from fastapi import APIRouter, Body, Depends, HTTPException
@@ -38,4 +39,7 @@ def get_ensemble(
             "started_at": ensemble.started_at,
         },
         size=ensemble.ensemble_size,
+        realization_storage_states=Counter(
+            state for states in ensemble.get_ensemble_state() for state in states
+        ),
     )

--- a/src/ert/dark_storage/json_schema/ensemble.py
+++ b/src/ert/dark_storage/json_schema/ensemble.py
@@ -4,6 +4,8 @@ from uuid import UUID
 
 from pydantic import BaseModel
 
+from ert.storage.realization_storage_state import RealizationStorageState
+
 
 class _Ensemble(BaseModel):
     size: int
@@ -19,3 +21,4 @@ class EnsembleOut(_Ensemble):
     id: UUID
     experiment_id: UUID | None = None
     userdata: Mapping[str, Any]
+    realization_storage_states: Mapping[RealizationStorageState, int] | None = None

--- a/src/ert/gui/tools/plot/plot_api.py
+++ b/src/ert/gui/tools/plot/plot_api.py
@@ -20,6 +20,7 @@ from resfo_utilities import history_key
 
 from ert.config import ParameterMetadata, ResponseMetadata
 from ert.services import StorageService
+from ert.storage.realization_storage_state import RealizationStorageState
 
 logger = logging.getLogger(__name__)
 
@@ -92,18 +93,27 @@ class PlotApi:
                             f"/ensembles/{ensemble_id}", timeout=self._timeout
                         )
                         self._check_response(response)
-                        response_json = response.json()
+                        response_json: dict[str, Any] = response.json()
                         ensemble_name: str = response_json["userdata"]["name"]
                         experiment_name: str = response_json["userdata"][
                             "experiment_name"
                         ]
                         ensemble_started_at = response_json["userdata"]["started_at"]
+                        ensemble_undefined = False
+                        if realization_storage_states := response_json.get(
+                            "realization_storage_states"
+                        ):
+                            ensemble_undefined = (
+                                RealizationStorageState.PARAMETERS_LOADED
+                                not in set(realization_storage_states)
+                            )
                         self._all_ensembles.append(
                             EnsembleObject(
                                 name=ensemble_name,
                                 id=ensemble_id,
                                 experiment_name=experiment_name,
-                                hidden=ensemble_name.startswith("."),
+                                hidden=ensemble_name.startswith(".")
+                                or ensemble_undefined,
                                 started_at=ensemble_started_at,
                             )
                         )

--- a/tests/ert/unit_tests/gui/tools/plot/conftest.py
+++ b/tests/ert/unit_tests/gui/tools/plot/conftest.py
@@ -87,29 +87,47 @@ def mocked_requests_get(*args, **kwargs):
 
     ensemble = {
         "/ensembles/ens_id_1": {
-            "name": "ensemble_1",
-            "experiment_name": "experiment",
-            "started_at": "2012-12-10T00:00:00",
+            "userdata": {
+                "name": "ensemble_1",
+                "experiment_name": "experiment",
+                "started_at": "2012-12-10T00:00:00",
+            }
         },
         "/ensembles/ens_id_2": {
-            "name": ".ensemble_2",
-            "experiment_name": "experiment",
-            "started_at": "2012-12-10T01:00:00",
+            "userdata": {
+                "name": ".ensemble_2",
+                "experiment_name": "experiment",
+                "started_at": "2012-12-10T01:00:00",
+            }
         },
         "/ensembles/ens_id_3": {
-            "name": "default_0",
-            "experiment_name": "experiment",
-            "started_at": "2012-12-10T02:00:00",
+            "userdata": {
+                "name": "default_0",
+                "experiment_name": "experiment",
+                "started_at": "2012-12-10T02:00:00",
+            }
         },
         "/ensembles/ens_id_4": {
-            "name": "default_1",
-            "experiment_name": "experiment",
-            "started_at": "2012-12-10T03:00:00",
+            "userdata": {
+                "name": "default_1",
+                "experiment_name": "experiment",
+                "started_at": "2012-12-10T03:00:00",
+            }
         },
         "/ensembles/ens_id_5": {
-            "name": "default_manyobs",
-            "experiment_name": "experiment",
-            "started_at": "2012-12-10T04:00:00",
+            "userdata": {
+                "name": "default_manyobs",
+                "experiment_name": "experiment",
+                "started_at": "2012-12-10T04:00:00",
+            }
+        },
+        "/ensembles/ens_id_uninitialized": {
+            "userdata": {
+                "name": "uninitialized_ensemble",
+                "experiment_name": "experiment",
+                "started_at": "2012-12-10T04:00:00",
+            },
+            "realization_storage_states": {"UNDEFINED": 1},
         },
     }
 
@@ -316,6 +334,7 @@ def mocked_requests_get(*args, **kwargs):
                 "ens_id_3",
                 "ens_id_4",
                 "ens_id_5",
+                "ens_id_uninitialized",
             ],
             "parameters": {
                 "default": [
@@ -379,7 +398,7 @@ def mocked_requests_get(*args, **kwargs):
     ]
 
     if args[0] in ensemble:
-        return MockResponse({"userdata": ensemble[args[0]]}, 200)
+        return MockResponse(ensemble[args[0]], 200)
     elif args[0] in observations:
         return MockResponse(
             observations[args[0]],

--- a/tests/ert/unit_tests/gui/tools/plot/test_plot_api.py
+++ b/tests/ert/unit_tests/gui/tools/plot/test_plot_api.py
@@ -114,10 +114,11 @@ def test_case_structure(api):
         "default_0",
         "default_1",
         "default_manyobs",
+        "uninitialized_ensemble",
     ]
 
     assert ensembles == expected
-    assert hidden_case == [".ensemble_2"]
+    assert hidden_case == [".ensemble_2", "uninitialized_ensemble"]
 
 
 def test_can_load_data_and_observations(api):

